Library/Homebrew/tap_constants.rb

```diff
@@ -5,7 +5,7 @@
 HOMEBREW_TAP_FORMULA_NAME_REGEX = T.let(/(?<name>[\w+\-.@]+)/, Regexp)
 # Match taps' formulae, e.g. `someuser/sometap/someformula`.
 HOMEBREW_TAP_FORMULA_REGEX = T.let(
-  %r{\A(?<user>[\w-]+)/(?<repo>[\w-]+)/#{HOMEBREW_TAP_FORMULA_NAME_REGEX.source}\Z},
+  %r{\A(?<user>[^/]+)/(?<repo>[^/]+)/#{HOMEBREW_TAP_FORMULA_NAME_REGEX.source}\Z},
```

Comment:
Thanks for the quick fix but: why do we need to permanently widen the range of acceptable tap names to fix this regression?

Reply:
@reitermarkus Ok. I'd like in future changes like this that are not strictly a fix but an actual functionality change to have some discussion before a `critical` self-merge, thanks.

Reply:
It doesn't change functionality though, except for making `Tap::from_path` consistent with what we actually allow in other places. Note that e.g. `brew install some/tap.with.dots/myformula` always worked, we just didn't use these regexes consistently in `Formulary` before.